psl/psl/tests/common/asserts.rs

```diff
@@ -622,9 +636,24 @@ impl DefaultValueAssert for ast::Expression {
 
     #[track_caller]
     fn assert_cuid(&self) -> &Self {
-        assert!(
-            matches!(self, ast::Expression::Function(name, args, _) if name == "cuid" && args.arguments.is_empty())
-        );
+        assert!(matches!(self, ast::Expression::Function(name, _, _) if name == "cuid"));
+
+        self
+    }
+
+    #[track_caller]
+    fn assert_cuid_version(&self, version: u8) -> &Self {
+        self.assert_cuid();
+
+        if let ast::Expression::Function(_, args, _) = self {
+            if let ast::Expression::NumericValue(actual, _) = &args.arguments[0].value {
+                assert_eq!(actual, &format!("{version}"));
+            } else {
+                panic!("Expected a numeric value for the version.");
+            }
+        } else {
+            unreachable!();
```

Comment:
Let's have a more descriptive panic message here

Reply:
`assert_cuid()` already runs `assert!(matches!(self, ast::Expression::Function(name, _, _) if name == "cuid"));`, so `unreachable!()` is quite accurate in this case. I'll add a more accurate description anyway for future-proofing

Reply:
https://github.com/prisma/prisma-engines/pull/5047/commits/0af3191a48f56508db206dbb15f614ce5c2277c3